public/javascripts/Gallery/src/filter/CardFilter.js

```diff
@@ -116,18 +116,22 @@ function CardFilter(uiCardFilter, labelTypeMenu, cityMenu, initialFilters) {
 
         // For each type of filter, check if it matches the default. If it doesn't, add to URL in a query param.
         if (status.currentLabelType !== 'Assorted') {
+            $('#clear-filters').show();
             newUrl += `?labelType=${status.currentLabelType}`;
             // Can only have applied tags if there is a specific label type chosen.
             if (currAppliedTags.length > 0) {
+                $('#clear-filters').show();
```

Comment:
Since we are showing the button any time the label type has changed, looks like you don't need to add the `show()` again here!